azure-pipelines/builds/ci.yml

```diff
@@ -15,18 +15,11 @@ stages:
 - stage: build
   displayName: Build
   jobs:
-  - template: /eng/common/templates/jobs/jobs.yml
-    parameters:
-      enablePublishUsingPipelines: true
-      enablePublishBuildArtifacts: true
-      enablePublishBuildAssets: true
-      artifacts:
-        publish:
-          artifacts: true
-          manifests: true
-      runSourceBuild: true
-      sourceBuildParameters:
-        includeDefaultManagedPlatform: true
+- template: /eng/common/templates/job/source-build.yml
```

Comment:
If you utilize the jobs template, you can make use of its parameter defaults.  All you would need is one line.

`- template: /eng/common/templates/jobs/source-build.yml`

Reply:
Fixed.  Thanks!